src/connection/serverComponent.ts

```diff
@@ -112,8 +112,20 @@ export class ServerComponent {
 
             ServerComponent.writeOutput(JSON.stringify({remotePath, ExecutablePathDir}));
 
+            const createDirResult = await connection.sendCommand({
+              command: `mkdir -p ${ExecutablePathDir} && chmod 500 ${ExecutablePathDir}`
```

Comment:
Why not 700?  Doesn't the lack of write permission prevent uploading the jar?

Reply:
I believe not since every user uploads the server component to their home directory, so in theory they own it.

@ThePrez care to chime in?

Reply:
I ran a test using `chmod 500` and I could still upload and write to a new file in the directory.  So, I guess it's ok.